packages/wezterm-bin/wezterm-bin.pacscript

```diff
@@ -0,0 +1,45 @@
+name="wezterm-bin"
+pkgname="wezterm"
+gives="wezterm"
+version="20230326-111934-3666303c"
+description="A GPU-accelerated cross-platform terminal emulator and multiplexer written by @wez and implemented in Rust"
+url="https://github.com/wez/wezterm/releases/download/${version}/wezterm-${version}.Ubuntu22.04.tar.xz"
+hash="636428e0faabd230f3be931131e47c708b1e0451ccce7a80e281346923c9b089"
+breaks="${pkgname}-app"
+repology=("project: wezterm")
+maintainer="smokeythemonkey <smokeythemonkey@posteo.net>"
+
+install() {
+  # Install Binaries
+  sudo install -Dm755 "${SRCDIR}/wezterm/usr/bin/wezterm" -t "${pkgdir}/usr/bin/"
+  sudo install -Dm755 "${SRCDIR}/wezterm/usr/bin/open-wezterm-here" "${pkgdir}/usr/bin/"
+  sudo install -Dm755 "${SRCDIR}/wezterm/usr/bin/strip-ansi-escapes" "${pkgdir}/usr/bin/"
+  sudo install -Dm755 "${SRCDIR}/wezterm/usr/bin/wezterm-gui" "${pkgdir}/usr/bin/"
+  sudo install -Dm755 "${SRCDIR}/wezterm/usr/bin/wezterm-mux-server" "${pkgdir}/usr/bin/"
+
+  # Copy Icons and plugins
+  sudo cp -r "${SRCDIR}/wezterm/usr/share/"* "/usr/share/"
+
+  # Install .desktop file
+  sudo install -Dm644 "${SRCDIR}/wezterm/usr/share/applications/org.wezfurlong.wezterm.desktop" "${pkgdir}/usr/share/applications/org.wezfurlong.wezterm.desktop"
+}
+
+postinst() {
+  sudo chmod +x "${SRCDIR}/wezterm/etc/profile.d/wezterm.sh"
```

Comment:
Why are you using `SRCDIR` outside of `{prepare,build,install}`?

Reply:
I tried pkgdir stowdir and stgdir, none of these worked. When looking for where the zip is downloaded and extracted to, SRCDIR was the only location where the files were available.

Reply:
i guess i could install the file first then run it....